apps/api/src/app/subscribers/e2e/update-global-preference.e2e.ts

```diff
@@ -77,7 +77,14 @@ describe('Update Subscribers global preferences - /subscribers/:subscriberId/pre
     expect(response.data.data.preference.channels).to.eql({});
   });
 
-  it('should update user global preference and disable the flag for the future channels update', async function () {
+  it('should unset all preferences when the preferences object is empty', async function () {
+    const response = await updateGlobalPreferences({}, session);
+
+    expect(response.data.data.preference.channels).to.eql({});
+  });
```

Comment:
This test case asserts that empty objects can be supplied when updating global preferences in relation to [this `UpsertPreferences` change](https://github.com/novuhq/novu/pull/6889/files#diff-bccd184dcf09c9cf0b60d2edcdcfac879d8f678d9b8655c8e471ff6fc3f3db0bR56)